client/src/outside/components/AssignmentsListing.vue

```diff
@@ -64,7 +67,9 @@ export default {
   methods: {
     rowClass: function (item) {
       let selected = item._id === this.$route.params.assignmentId
-      return selected ? 'selected' : ''
+      let classString = selected ? 'selected ' : ''
+      const isWarning = this.isItemMisconfigured(item) ? 'mis-configuration' : ''
```

Comment:
Let's change the wording from mis-configured to un-configured.  The item has a valid configuration it's just not yet configured by a user.